### PR TITLE
Make bpmn quickstart generic

### PR DESCRIPTION
**Pull request by MaxTru**
* Given Web and Desktop Modeler co-existence, the quickstart shall be
  agnostic
* So I removed all desktop modeler specifics. IMO the quickstart is still understandable and has the key contents included.

FYI: @CatalinaMoisuc

### CODE REVIEW

File: docs/components/modeler/bpmn/bpmn.md [hunk 1]
```diff
@@ -4,13 +4,7 @@ title: BPMN in Modeler
...
-
-To start modeling, create a BPMN 2.0 diagram by selecting **Create New Diagram** in the **Diagrams** tab for your process engine in the top-level menu.
-
-:::note
-If working in Camunda Platform 7, select **Create diagram > Create new BPMN diagram**.
-:::
+# Model a BPMN 2.0 diagram
 
 :::note
 BPMN diagrams must be created for the process engine they intend to be deployed on. You cannot run a BPMN diagram modeled for Camunda Platform 7 in Camunda Cloud, or vice versa, at this time.
```
Comment (akeller) [inline docs/components/modeler/bpmn/bpmn.md:10]:
```suggestion
BPMN diagrams must be created for the process engine they intend to be deployed on. You cannot run a BPMN diagram modeled for Camunda Platform 7 in Camunda Platform 8, or vice versa, at this time.
```
```suggestion
BPMN diagrams must be created for the process engine they intend to be deployed on. You cannot run a BPMN diagram modeled for Camunda Platform 7 in Camunda Cloud, or vice versa, at this time.
```

Comment (akeller) [inline docs/components/modeler/bpmn/bpmn.md:10]:
```suggestion
BPMN diagrams must be created for the process engine they intend to be deployed on. You cannot run a BPMN diagram modeled for Camunda Platform 7 in Camunda Platform 8, or vice versa, at this time.
```
```suggestion
BPMN diagrams must be created for the process engine they intend to be deployed on. You cannot run a BPMN diagram modeled for Camunda Platform 7 in Camunda Cloud, or vice versa, at this time.
```

**Comment by MaxTru** [comment]
Good catch! Adjusted. Thanks

**Review by akeller — APPROVED** [review]
LGTM!

**Pull request merged**